src/server/api/routers/dns-hole/router.ts

```diff
@@ -131,7 +133,13 @@ const processPiHole = async (app: ConfigAppType, enable: boolean) => {
 
 const collectPiHoleSummary = async (app: ConfigAppType) => {
   const piHole = new PiHoleClient(app.url, findAppProperty(app, 'apiKey'));
-  const summary = await piHole.getSummary();
+  const summary = await piHole.getSummary().catch(() => {
+    return null;
+  });
+
+  if (!summary) {
+    return null;
+  }
```

Comment:
Why not throw an error ? Won't this just display nothing ?

Reply:
Because throwing an error blocks the usage of the widget in case there is a second instance of dns-hole.
Display nothing is true though, I'll make it the same as other widget where instead of being stuck on a load loop it'll show a text about no dns-holes found.